lib/simulator-xcode-6.js

```diff
@@ -100,16 +100,10 @@ class SimulatorXcode6 extends EventEmitter {
    * or, for iOS 7.1, the app name without `.app` (e.g., MobileSafari)
    */
   async getAppDir (id, subDir = 'Data') {
-    if (await this.isFresh()) {
-      log.info('Attempted to get an app path from a fresh simulator ' +
-               'quickly launching the sim to populate its directories');
-      await this.launchAndQuit();
```

Comment:
I don't understand. In order to make changes to app directory files (e.g., to set certain flags for the purpose of say the `locationServicesEnabled` capability), we have to ensure that the app has previously been launched. In our previous experimentation, setting the flag in a running sim did not take effect; we had to launch the app, modify the plist, then shut down the sim in order for it to reliably work. It looks like you're removing this behavior?